src/Core/Framework/Store/Services/StoreClient.php

```diff
@@ -34,6 +36,8 @@
 class StoreClient
 {
     public const EXTENSION_LICENSE_IS_ALREADY_CANCELLED = 'ShopwarePlatformException-61';
+    public const EXTENSION_LIST_CACHE = 'extensionListStatus';
+    public const EXTENSION_LIST_TTL = 3600; // 1 hour
```

Comment:
I think, one hour doesn't really help the SBP, right? They were complaining about "several times a day", right?